modules/core_project_factory/main.tf

```diff
@@ -100,13 +100,18 @@ module "project_services" {
 /******************************************
   Shared VPC configuration
  *****************************************/
+resource "time_sleep" "wait_5_seconds" {
```

Comment:
From the issue it seems like we only need this if `var.vpc_service_control_attach_enabled`?
Might be good to make this a conditional creation via `var.vpc_service_control_attach_enabled ? 1 : 0`

Reply:
@bharathkkb yes sure good point, ill push the change